package.json

```diff
@@ -1,67 +1,71 @@
 {
   "name": "ember-engines-website",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "private": true,
   "description": "The official website for Ember Engines",
   "repository": "https://github.com/ember-engines/ember-engines.com",
   "license": "MIT",
   "author": "",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "keywords": [
+    "ember-addon"
+  ],
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
+  },
+  "dependencies": {
+    "ember-cli-babel": "^7.18.0",
+    "ember-cli-htmlbars": "^4.2.3"
   },
   "devDependencies": {
-    "@ember/jquery": "^0.6.0",
-    "@ember/optional-features": "^0.7.0",
+    "@ember/optional-features": "^1.3.0",
+    "@glimmer/component": "^1.0.0",
+    "@glimmer/tracking": "^1.0.0",
+    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-ajax": "^5.0.0",
-    "ember-auto-import": "^1.2.21",
-    "ember-cli": "~3.12.0",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.7.3",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
-    "ember-cli-fastboot": "^2.1.3",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-sass": "^10.0.0",
-    "ember-cli-shims": "^1.2.0",
-    "ember-cli-showdown": "^4.4.4",
+    "ember-auto-import": "^1.5.3",
+    "ember-cli": "~3.13.1",
+    "ember-cli-addon-docs": "^0.6.16",
+    "ember-cli-addon-docs-esdoc": "^0.2.3",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-deploy": "^1.0.2",
```

Comment:
how can we manage to deploy here? As I know this codebase already deploys on master when CI is green. 

If we going to change  it, maybe we need rethink our infrastructure on Heroku

Reply:
This will currently deploy to gh-pages branch, do we deploy from that?

Reply:
Currently, we are doing deploy automatically to Heroku. I prefer GitHub pages, but we need to change it and make sure that it will work fine without some unavailability. And make sure that GitHub pages will have current sit URL/domain.

cc: @dgeb